Update Logos

larson-carter opened the pull request:
I updated our logos

Commit:
Update Logos

I updated our logos

BaffourAdu commented:
Nice work @larson-carter! 

I would want to point out that the logo on the dashboard in `About.vue` isn't showing also suggest the favicon should have a transparent background.

Commit:
Fix Favicon and Dashboard Logo

I made the Favicon transparent

I also rerouted the Dasboard logo to be in the proper directory!

BaffourAdu reviewed (approved):
Looks good!
